service/assessment/assessment.go

```diff
@@ -96,7 +96,9 @@ type Service struct {
 	// hookMutex is used for (un)locking result hook calls
 	hookMutex sync.RWMutex
 
-	// Currently, results are just stored as a map (=in-memory). In the future, we will use a DB.
+	// results will contain all the assessment results of the current run. They will NOT be persisted to the storage,
```

Comment:
```suggestion
	// Results will contain all the assessment results of the current run. They will NOT be persisted to the storage,
```

Reply:
What I forgot: I think you have to update the engine as well.

Reply:
results is unexported -> lower case